build.sbt

```diff
@@ -24,30 +38,16 @@ inThisBuild(Seq(
     "-Xlint:_",
   ),
   scalacOptions ++= {
-    if (CrossVersion.partialVersion((udash / scalaVersion).value).contains((2, 12))) Seq(
+    if (scalaBinaryVersion.value == "2.12") Seq(
       "-Ywarn-unused:_,-explicits,-implicits",
       "-Ybackend-parallelism", "4",
       "-Ycache-plugin-class-loader:last-modified",
       "-Ycache-macro-class-loader:last-modified"
     ) else Seq.empty
   },
-  resolvers += Resolver.defaultLocal
-))
-
-val forIdeaImport = System.getProperty("idea.managed", "false").toBoolean && System.getProperty("idea.runid") == null
-val CompileAndTest = "test->test;compile->compile"
-val TestAll = "test->test"
-
-// Settings for JS tests run in browser
-val browserCapabilities: Capabilities = {
-  // requires ChromeDriver: https://sites.google.com/a/chromium.org/chromedriver/
-    val options = new ChromeOptions()
-    options.addArguments("--headless", "--disable-gpu")
-    options
-}
-
-val commonSettings = Seq(
   moduleName := "udash-" + moduleName.value,
```

Comment:
Since we're changing the modules structure anyway, I think we could remove `udash-` prefix. What do you think?

Reply:
Hmm, jars would be named like `core-x.y.z.jar` which might easily conflict with whatever other library is using the same convention (remember that UMP puts all jars into one folder).